rand_chacha/src/chacha.rs

```diff
@@ -114,7 +114,7 @@ macro_rules! chacha_impl {
         /// except that we use a stream identifier in place of a nonce. A 64-bit counter over 64-byte
         /// (16 word) blocks allows 1 ZiB of output before cycling, and the stream identifier allows
         /// 2<sup>64</sup> unique streams of output per seed. Both counter and stream are initialized
-        /// to zero but may be set via [`set_word_pos`] and [`set_stream`].
+        /// to zero but may be set via the `set_word_pos` and `set_stream` methods.
```

Comment:
Why are they no longer linked?

Reply:
See commit notes. The links didn't work due to lack of macro substitution.